### PR TITLE
Results data table

### PR DESCRIPTION
**Pull request by sashamaryl**
addresses issue #179  
Update Shared Data Table UI Component

### CODE REVIEW

File: frontend/tests/snapshots/search.test.tsx [hunk 1]
```diff
@@ -7,8 +7,11 @@ jest.mock("../../compositions/visualizations/map")
 beforeAll(() => setAuthForTest())
 
 it("renders Dashboard correctly", async () => {
+  // Ensure consistent title id's for fontawesome
```
Comment (alexjball) [inline frontend/tests/snapshots/search.test.tsx:10]: The tests still seem to pass with these lines removed. Could you revert the changes in this file?

**Review by alexjball — APPROVED** [review]
LGTM!

File: frontend/tests/snapshots/search.test.tsx (hunk 1 above)
Comment (sashamaryl) [reply frontend/tests/snapshots/search.test.tsx]: resolved in db449aa

**Review by alexjball — APPROVED** [review]
